ofrak_core/ofrak/gui/server.py

```diff
@@ -1193,8 +1193,11 @@ async def get_project_script(self, request: Request) -> Response:
         script_name_query = request.query.get("script")
         if script_name_query is not None:
             script_name = script_name_query
-        project = self._get_project_by_id(project_id)
-        script_body = project.get_script_body(script_name)
+        if script_name == "undefined":
```

Comment:
Would it be the string "undefined" or would it be deserialized to `None` in Python?

Reply:
Comes from javascript